modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -35,7 +35,7 @@ metadata:
   name: "clusterlogging"
 ...
 nodes:
-  - nodeCount: 2 <1>
+  - nodeCount: 3 <1>
```

Comment:
@mburke5678  There are some limitation about the Elasticsearch node number when using Cluster Logging custom Resource (CR). 
1) The CLO only create ES nodes using "roles": ["client","data","master"]. 
2) The nodeCount should be <=3.     as Elasticsearch Operator don't allow the ES master nodes number > 3

To customize Elasticsearch nodes type. Turn to CLO Unmanage status, and modify the elasticsearch CR directly
For example:  Add two dataonly ES node to cluster. You can use the following Elasticsearch CR
apiVersion: logging.openshift.io/v1alpha1
kind: Elasticsearch
metadata:
  name: elasticsearch
  namespace: openshift-logging
spec:
  managementState: Managed
  nodeSpec:
    image: quay.io/openshift/origin-logging-elasticsearch5:latest
  nodes:
  - nodeCount: 2
    roles:
    - client
    - data
    - master
    storage:
      size: 10Gi
      storageClassName: gp2
  - nodeCount: 2
    roles:
    - data
    storage:
      size: 10Gi
      storageClassName: gp2
  redundancyPolicy: ZeroRedundancy

Reply:
@anpingli Will the default memory settings allow for adding two data-only nodes, as you sow here? (I would test this myself. But, I am not running the default memory and the pods were not getting created due to insufficient memory.)